migrator/migrator.proto

```diff
@@ -27,14 +27,15 @@ option java_outer_classname = "MigratorProto";
 package migrator;
 
 service Migrator {
-    rpc ExportData(Export.Req) returns (stream Export.Progress);
-    rpc ImportData(Import.Req) returns (stream Import.Progress);
+    rpc ExportDatabase(Export.Req) returns (stream Export.Progress);
+    rpc ImportDatabase(Import.Req) returns (stream Import.Progress);
 }
 
 message Export {
     message Req {
         string database = 1;
-        string filename = 2;
+        string data_file = 2;
```

Comment:
useful to maintain backwards compat so users can still use the same migrator to import into the new database on a different TypeDB server

Reply:
But it's a shame that this reverses the order of schema and data compared to everywhere else.